source/postprocess/gravity_point_values.cc

```diff
@@ -52,10 +52,22 @@ namespace aspect
       output_file_number (numbers::invalid_unsigned_int)
     {}
 
+
     template <int dim>
     std::pair<std::string,std::string>
     GravityPointValues<dim>::execute (TableHandler &)
     {
+      AssertThrow(false, ExcNotImplemented());
+      return std::pair<std::string,std::string>();
+    }
+
+
+    template <>
+    std::pair<std::string,std::string>
+    GravityPointValues<3>::execute (TableHandler &)
+    {
+      const int dim = 3;
```

Comment:
`const unsigned int`

Reply:
Interesting point. We normally use unsigned ints, but our template parameters are always
```
template <int dim>
```
It looks like we mostly use signed ints in ASPECT in this situation, but ~75% unsigned inside deal.II:
```
$ grep "const int dim" -r deal-git/source/ | wc -l
25
$ grep "const unsigned int dim" -r deal-git/source/ | wc -l
87
```
I will summon @bangerth for his opinion. :-)